src/Components/ADT3DSceneBuilder/Internal/Shared/MakePivotRequired.tsx

```diff
@@ -0,0 +1,26 @@
+import { IPivotItemProps } from '@fluentui/react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { customPivotItemStyles } from '../Behaviors/BehaviorsForm.styles';
+
+export function setPivotToRequired(
```

Comment:
It may be better to place this function in a place outside of ADT3DSceneBuilder just because the Pivot is a component that we stand to use in places other than just 3D Scenes.  That is, a UI for making a pivot appear required is a concept that is something we are likely to want to use across cardboard, not just in 3D Scenes.  My personal preference is src/Theming/FluentComponentStyles/Pivot.styles.ts, but open to other options.  Regardless, I think it should probably move to a place not restricted to 3D Scenes